### ISSUE TITLE
Bitrise build is failing on Ubuntu

### ISSUE THREAD

**Issue by gabrielweyer** [cake-build/cake#2033]
It seems like the `Bitrise` `Ubuntu` build has been failing since the 21st of Jan.

I tried to investigate but the [logs](https://www.bitrise.io/build/b82041e8aaf8d224) are not public:

![image](https://user-images.githubusercontent.com/2101647/35761397-0bbea09e-08dc-11e8-877e-32f18aad52ba.png)

**Comment by gabrielweyer** [comment]
Doesn't seem to be a `Ubuntu` issue. I created a `Ubuntu 16.04` Azure VM and ran a successful build on it with the following version of `Mono`:

```bash
Mono JIT compiler version 5.8.0.108 (tarball Fri Jan 19 18:15:21 UTC 2018)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       altstack
        Notifications: epoll
        Architecture:  amd64
        Disabled:      none
        Misc:          softdebug
        LLVM:          supported, not enabled.
        GC:            sgen (concurrent by default)
```

**Comment by devlead** [comment]
Yes we know that, we've upgraded our Bitrise image to .NET Core 2.1.4, as that's what's going to be used coming cake releases, but current development is on 1.1.x.

That's why the PR #1812 isn't failing, which basically is what will be the 0.26.0 release.

**Comment by augustoproiete** [comment]
Closing this as resolved in the v0.30.0 milestone as part of the v0.30.0 release.
